mulighetsrommet-api/src/main/kotlin/no/nav/mulighetsrommet/api/refusjon/db/ForslagRepository.kt

```diff
@@ -0,0 +1,102 @@
+package no.nav.mulighetsrommet.api.refusjon.db
+
+import kotlinx.serialization.encodeToString
+import kotlinx.serialization.json.Json
+import kotliquery.Row
+import kotliquery.Session
+import kotliquery.queryOf
+import no.nav.mulighetsrommet.database.Database
+import no.nav.mulighetsrommet.domain.dto.amt.Forslag
+import org.intellij.lang.annotations.Language
+import java.util.*
+
+class ForslagRepository(private val db: Database) {
```

Comment:
Kan vi tydeliggjøre at  dette er for deltakere? Både klasse og databasetabell?

Reply:
deltaker_forslag f.eks.

Reply:
👍